ci/Jenkinsfile

```diff
@@ -132,13 +164,44 @@ pipeline {
                         }
                     }
                 }
-
             }
         }
         stage('Tests') {
             failFast true
             parallel {
-                stage('Test') {
+                stage('TestDebug') {
+                     options {
+                         timeout(time: 1, unit: 'HOURS')
+                     }
+                     agent {
+                         docker {
+                             image 'gpuci/rapidsai:21.10-cuda11.4-devel-ubuntu20.04-py3.8'
+                             label 'driver-495'
+                             args '--cap-add=sys_nice'
+                         }
+                     }
+                     environment {
+                         BUILD_TYPE = "Debug"
+                         HOME = "${WORKSPACE}"
```

Comment:
I did something a little different here and switched on BUILD_TYPE, since the tests will just be run with a debug build. Then in test.sh run the coverage report and export the artifacts if the build type is Debug.